### PR TITLE
Update copy and add empty state messages

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

**Comment by znarf** [comment]
screen shot 2018-08-31 at 13 39 31

screen shot 2018-08-31 at 13 40 03

screen shot 2018-08-31 at 13 40 21

**Comment by xdamman** [comment]
Looks great. Ship it!

File: src/components/DependencyTable.js [hunk 1]
```diff
@@ -67,49 +69,61 @@ export default class DependencyTable extends React.Component {
...
-                      </a>
-                    </Fragment>
-                  }
-                </td>
+        {dependencies.length === 0 &&
+          <div className="error">
+            <p>Sorry, we could not detect any dependencies. </p>
+            <p>We currently support JavaScript (NPM), PHP (Composer), .NET (Nuget) and Go (dep).</p>
+            <p>
+              Want so see something else? <Link route="contributing"><a>Contribute</a></Link>.
```
Comment (piamancini) [inline src/components/DependencyTable.js]: s/Contribute for You can contribute here.

File: src/components/RecommendationList.js [hunk 2]
```diff
@@ -24,7 +28,20 @@ export default class RecommendationList extends React.Component {
...
-          {recommendations.filter(r => r.opencollective).map(recommendation => (
+          {projectsRequiringFunding.length === 0 &&
+            <div className="error">
+              <p>Sorry, we could not detect any projects requiring funding. We currently:</p>
+              <ul>
+                <li>detect dependencies from JavaScript (NPM), PHP (Composer), .NET (Nuget) and Go (dep).</li>
+                <li>match them with projects registered on <a href="https://opencollective.com/">Open Collective</a></li>
+              </ul>
+              <p>
+                Want so see something else? <Link route="contributing"><a>Contribute</a></Link>.
```
Comment (piamancini) [inline src/components/RecommendationList.js]: idem

File: src/pages/index.js [hunk 3]
```diff
@@ -149,8 +156,13 @@ export default class Index extends Component {
...
           <p className="description">
-            Discover the open source projects your organization
-            is using that need financial support.
+              Discover the Open Source projects<small><sup>*</sup></small> your organization
+              is using that need financial support.
+          </p>
+
+          <p className="description secondaryDescription">
+            * We currently detect dependencies from JavaScript (NPM), PHP (Composer), .NET (Nuget) and Go (dep).<br />
+            Want so see something else? <Link route="contributing"><a>Contribute</a></Link>.
```
Comment (piamancini) [inline src/pages/index.js]: idem

**Comment by znarf** [comment]
@piamancini What about "See how to contribute"

**Comment by piamancini** [comment]
@znarf go for it

**Pull request merged**